lib/bitters/generator.rb

```diff
@@ -16,11 +16,12 @@ def install
         puts "Bitters files already installed, doing nothing."
       else
         install_files
-        puts "Bitters files installed to #{install_path}/base"
+        puts "Bitters files installed to #{install_path}/"
```

Comment:
Not really, as the path would be `custom_path/base`, due to https://github.com/thoughtbot/bitters/blob/8e0040297999ae44cecce91cee7d9e52b71b4e54/lib/bitters/generator.rb#L56.
This diff line (from the 2nd commit), only fixes the CLI output when running for example `bitters install --path foo`. It outputs `Bitters files installed to foo/base/base` when it should show `foo/base`.